run.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+set -e
+
+VERSION='0.1'
+
+DOCKER_COMPOSE=$(which docker-compose)
+BASE_COMPOSE='/etc/moodle-docker/docker-compose.yml'
+THEME_COMPOSE='/etc/moodle-docker/dc.theme-dev.yml'
+BASE_COMMAND="${DOCKER_COMPOSE} --file ${BASE_COMPOSE}"
+
+THEME_DIR='/opt/moodle/theme'
+
+# Print usage and argument list
+function print_usage {
+cat << EOF
+usage: ${0} options
+
+This script will create an organization and update the associated DNS record.
+
+OPTIONS:
+Actions:
+    clean
+    down
+    stop
+    restart
+    up
+    version, -v, --version  Print ${0} version and exit
+
+Parameters:
+    -d      Detach from running docker-compose containers
+    -t      Mount theme directory to host for theme development
+
+Examples:
+"${0} start -d -t" Start Moodle in detached mode with theme directory mounted to host.
+
+Version: ${VERSION}
+EOF
+}
+
+function ensure_theme_dir {
+    if [ ! -d "${THEME_DIR}" ] ; then
+        echo "Theme directory not found, creating ${THEME_DIR}"
+        mkdir -p "${THEME_DIR}"
+    fi
+}
+
+function clean_moodle {
+    local compose_command="${BASE_COMMAND}"
+
+    if [ "${THEME_DEV}" == 'True' ] ; then
+        compose_command="${compose_command} --file ${THEME_COMPOSE}"
+    fi
+    compose_command="${compose_command} rm"
+
+    eval "${compose_command}"
+}
+
+function down_moodle {
+    local compose_command="${BASE_COMMAND}"
+
+    if [ "${THEME_DEV}" == 'True' ] ; then
+        compose_command="${compose_command} --file ${THEME_COMPOSE}"
+    fi
+    compose_command="${compose_command} down"
+
+    eval "${compose_command}"
+}
+
+function stop_moodle {
+    local compose_command="${BASE_COMMAND}"
+
+    if [ "${THEME_DEV}" == 'True' ] ; then
```

Comment:
Can probably abstract out this `if` block into a function and reuse it the other places with the same code